### PR TITLE
#77 - Add unit tests for CachedProxySlice

### PR DESCRIPTION
**Pull request by genryxy**
Part of #77 
Added unit tests for `CachedProxySlice` for different scenarios

### CODE REVIEW

File: src/test/java/com/artipie/composer/http/CachedProxySliceTest.java [hunk 1]
```diff
@@ -23,9 +23,165 @@
...
+        this.storage.save(key, new Content.From(cached)).join();
+        MatcherAssert.assertThat(
+            "Returns body from remote",
+            new CachedProxySlice(
+                new SliceSimple(
+                    new RsWithBody(StandardRs.OK, new Content.From(remote))
+                ),
+                new FromRemoteCache(this.storage)
+            ),
+            new SliceHasResponse(
```
Comment (olenagerasimova) [inline src/test/java/com/artipie/composer/http/CachedProxySliceTest.java:81]: @genryxy correct `SliceHasResponse` usage would be in this case:
```
new SliceHasResponse(
    Matchers.allOf(
        RsHasStatus(...), RsHasHeaders(...), RsHasBody(...)
    ),
    new RequestLine(RqMethod.GET, String.format("/%s", key.string()))
)
```
third and fourth ctor parameters are request related, not response. If you copied this assertion from somewhere, let's fix it there too.

File: src/test/java/com/artipie/composer/http/CachedProxySliceTest.java [hunk 2]
```diff
@@ -23,9 +23,165 @@
...
+        final Key key = new Key.From("key");
+        new BlockingStorage(this.storage).save(key, body);
+        MatcherAssert.assertThat(
+            "Returns body from cache",
+            new CachedProxySlice(
+                new SliceSimple(new RsWithStatus(RsStatus.INTERNAL_ERROR)),
+                new FromRemoteCache(this.storage)
+            ),
+            new SliceHasResponse(
+                new RsHasStatus(RsStatus.OK),
```
Comment (olenagerasimova) [inline src/test/java/com/artipie/composer/http/CachedProxySliceTest.java:131]: @genryxy same mistake here

**Review by olenagerasimova — CHANGES_REQUESTED** [review]
@genryxy two comments, take a look please

**Comment by genryxy** [comment]
@olenagerasimova thanks, corrected

**Review by olenagerasimova — APPROVED** [review]
@genryxy thanks, looks good to merge

**Comment by 0crat** [comment]
Job `gh:artipie/composer-adapter#95` is not assigned, can't get performer

**Pull request merged**